pkg/jobs/jobsauth/authorization_test.go

```diff
@@ -220,13 +220,12 @@ func TestAuthorization(t *testing.T) {
 			accessLevel: jobsauth.ViewAccess,
 		},
 		{
-			name:        "users-cannot-control-their-own-jobs",
+			name:        "users-can-control-their-own-jobs",
```

Comment:
while we're here: could you add the following case: "controljob-system-priviledge-_insufficient_-for-admin-jobs"

Reply:
Done.